tests/test_rpn.py

```diff
@@ -5,3 +5,6 @@ class TestRPN(TestCase):
 
     def test_rpn_eval(self):
         assert rpn.rpn_eval(["3" , "2", "*", "11" , "-"]) == -5.0
+
+    def test_rpn_powers(self):
```

Comment:
Treat these as unit tests [1]. A unit test is a test for the smallest possible unit of code. Typically your units should be functions/methods, unless your one function does a lot of things. You should try to avoid overloading methods with functionality, if possible, as a good practice. You already have unit tests to make sure that the operators function properly, so you don't need to have extra tests. Remember that a test can have multiple assert statements. So feel free to have a bunch in the `test_rpn_eval` test function!

[1] https://en.wikipedia.org/wiki/Unit_testing